app/src/main/java/com/tristanwiley/chatse/chat/ChatActivity.kt

```diff
@@ -65,6 +67,8 @@ class ChatActivity : AppCompatActivity(), ServiceConnection {
     private var isBound = false
     private val prefs = SharedPreferenceManager.sharedPreferences
 
+    private var room_num: Int? = -1
```

Comment:
Why is this variable nullable?

Reply:
this is a overlook on my part. will fix.

Reply:
Ok!